apps/condo/domains/settings/schema/MobileFeatureConfig.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyProgressionMeterReadingsIsEnabled:Checkbox; meta:Json; ticketSubmittingIsDisabled:Checkbox'`
+ */
+
+const { Text, Relationship, Checkbox } = require('@keystonejs/fields')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/settings/access/MobileFeatureConfig')
+
+
+
+const ERRORS = {
+    TICKET_SUBMITTING_PHONES_NOT_CONFIGURED: {
```

Comment:
do you need to specify `variable` here?

Reply:
No, I just wrote. Copy-pasted from another domain

Reply:
add `variable: ['data', 'commonPhone'],`